plugins/User/plugin.py

```diff
@@ -523,9 +524,15 @@ def auth(self, irc, msg, args, url):
                     'Authentication aborted.'), Raise=True)
             verified = gpg.keyring.verify(data)
             if verified and verified.valid:
-                keyid = verified.key_id
+                keyid0 = verified.key_id
+                fprint = verified.pubkey_fingerprint
+                kprint = fprint[-16:]
                 prefix, expiry = self._tokens.pop(token)
                 found = False
+                if keyid0 == kprint:
```

Comment:
Wouldn't it make more sense to do `keyid0.endswith(fprint)` instead of `keyid0 == kprint`?